laurat007/assignment2/AncestorsTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment_2;
+
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import junit.framework.TestCase;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Laura
+ */
+public class AncestorsTest extends TestCase{
```

Comment:
Generally looks good, well done :)

Can you maybe try to add a couple more tests? For getAncestors method and for common ancestor. Try to have at least 2 or 3 tests for each, and maybe test edge cases like when there is no node with a given value etc

Reply:
added more tests.